python/semantic_kernel/__init__.py

```diff
@@ -30,7 +29,6 @@
 
 __all__ = [
     "Kernel",
-    "NullLogger",
```

Comment:
let's not delete it here either, if people import direct or use sk.NullLogger.

Reply:
Fixed